static/partials/about.html

```diff
@@ -19,9 +19,9 @@ <h1 class="page-header">About
 
 
                 <h2>Educational program:</h2>
-                <p>Complete educational program for the whole year is accesable here: <a href="#/program">Program</a></p></p>
-                <p>We are still working on kickoff weekend workshops agenda<a href="#/agenda"></a>.</p>
-            </div>
+                <p>Complete educational program for the whole years is accesable here: <a href="#/program">Program</a></p></p>
+                <p>We are still working on kickoff weekend workshops we are still working<a href="#/agenda"></a></p>
```

Comment:
zostawić poprzednie raczej: `<p>Complete educational program for the whole year is accesable here: <a href="#/program">Program</a></p></p><p>We are still working on kickoff weekend workshops agenda<a href="#/agenda"></a>.</p>`